server/integration/infinispan/src/main/resources/subsystem-templates/infinispan-core.xml

```diff
@@ -141,7 +141,7 @@
          <cache-container name="local" default-cache="default">
             <global-state />
             <local-cache name="default">
-               <remote-store cache="default" socket-timeout="60000" tcp-no-delay="true" shared="true" hotrod-wrapping="true" purge="false" passivation="false">
+               <remote-store cache="default" socket-timeout="60000" tcp-no-delay="true" shared="true" hotrod-wrapping="true" protocol-version="2.0" purge="false" passivation="false">
```

Comment:
If we want to use remote iterators, the test would fail for servers which doesn't support HR 2.5 (IIRC implemented in ISPN 8.2, so fails also for 8.0). I'll probably changed it to 2.5 as this is used for generating examples (so that users are adviced to use 2.5 if they can) and generate config file via XSLT for tests, so that HR version can be overwritten by properties if needed